src/toolbars/TimeSignatureToolBar.cpp

```diff
@@ -162,7 +167,7 @@ void TimeSignatureToolBar::Populate()
       wxEVT_SPINCTRL,
       [this](auto)
       {
-         const auto upper = mUpperSignatureControl->GetValue();
+         const auto upper = int(mUpperSignatureControl->GetValue());
```

Comment:
static_cast?

Reply:
Well, this is not a c-cast and this form is much shorter. I don't have any hard feelings either